### ISSUE TITLE
schematic not found error in game

### ISSUE THREAD

**Issue by anovickis** [GreatOrator/TolkienCraft-3-Return-to-Middle-earth#83]
## Base information
* Minecraft version:
* Minecraft Forge version:
* TolkienCraft III version:
as provided by twitch tolkiencraft III mod pack

## Crash report
```
CRASH REPORT HERE
```
In game log:

[Error - 404] Schematic {} not found!!!

for 1 Faradrim court when using the key

**Comment by GreatOrator** [comment]
Can you take a look in the mod folder for a folder called townbuilder and see if you can find the schematic in there?

**Comment by Neocentric** [comment]
Same issue here for 1 Adenedhal Place.  Only thing in the townbuilder folder is wall.schematic

**Comment by GreatOrator** [comment]
Ok, so I did miss the schematics when I posted the update (did not realize they did not transfer over, plus was one of the few features I did not test but looks like I will have to re-set all of the blocks even with the schematics in the directory).

I will push a new pack update with the schematics, but will take time to redo all of the houses.  For now just break down the iron door as needed and I will update those as soon as I can.
